Pipfile

```diff
@@ -31,7 +31,7 @@ django-storages = {extras = ["google"],version = "*"}
 django-templated-email = ">=2.2.0"
 django-versatileimagefield = ">=1.10"
 django-webpack-loader = ">=0.3.0"
-django-elasticsearch-dsl = ">=0.5.0.<0.6.0"
+django-elasticsearch-dsl = ">=0.5.0,<0.6.0"
```

Comment:
Note for the reviewers: this was making pipenv trying to install the 6.4 version (0.6.4 -> 6.4.2, I assume) that is now depending on `elasticsearch-dsl >=6.4.0` where we explicitly ask for `>=6.0.0,<6.1.0`.

Not sure if we should upgrade it. But for sure, this is not what this PR is for.

Reply:
Not sure about it as well; I think @cmiacz had a problem with resolving this conflict in Pipenv yesterday.

Reply:
Yes, me too. It is very hard to find the cause, even by using `pipenv graph` to see the dependency graph. You cannot see the one we are trying to upgrade to, only the current one.